homeassistant/components/blink/sensor.py

```diff
@@ -51,13 +54,25 @@ def __init__(self, data, camera, description: SensorEntityDescription):
         self._attr_name = f"{DOMAIN} {camera} {description.name}"
         self.data = data
         self._camera = data.cameras[camera]
+        self._camera_name = camera
         self._attr_unique_id = f"{self._camera.serial}-{description.key}"
         self._sensor_key = (
             "temperature_calibrated"
             if description.key == "temperature"
             else description.key
         )
 
+    @property
+    def device_info(self):
```

Comment:
As above, please use dataclass and attr format: `self._attr_device_info = DeviceInfo(...)`